src/components/LetterBtn/LetterBtn.tsx

```diff
@@ -0,0 +1,27 @@
+import { MouseEvent } from 'react';
+import classNames from 'classnames';
+import styles from './letterBtn.module.scss';
+
+type LetterBtnProps = {
+  variant: string;
```

Comment:
To be more concise you can do `variant: "primary" | "secondary"`.
This way typescript does not allow any other values than those two.